homeassistant/components/brunt/config_flow.py

```diff
@@ -0,0 +1,89 @@
+"""Config flow for brunt integration."""
+import logging
+
+from aiohttp import ClientResponseError
+from aiohttp.client_exceptions import ServerDisconnectedError
+from brunt import BruntClientAsync
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {vol.Required(CONF_USERNAME): str, vol.Required(CONF_PASSWORD): str}
+)
+
+
+class BruntConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for youless."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    def __init__(self):
+        """Start the Brunt config flow."""
+        self._reauth_entry = None
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        if user_input is None:
+            return self.async_show_form(step_id="user", data_schema=DATA_SCHEMA)
+
+        errors = None
+        bapi = BruntClientAsync(
+            username=user_input[CONF_USERNAME], password=user_input[CONF_PASSWORD]
+        )
+        try:
+            await bapi.async_login()
+        except ClientResponseError as exc:
+            if exc.status == 403:
+                _LOGGER.warning("Brunt Credentials are incorrect")
```

Comment:
Kinda duplicate, its already in the UI
```suggestion
```

Reply:
Also for yaml config, or is the yaml passed to the config flow?

Reply:
In that case, the import fails and aborts, which I think is logged. That said, they already have had warnings/issues in that case.